sourcemap/decoder.py

```diff
@@ -197,6 +234,18 @@ def decode(self, source):
 
 # Mapping of base64 letter -> integer value.
 # This weird list is being allocated for faster lookups
-B64 = [-1] * 123
-for i, c in enumerate('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'):
-    B64[ord(c)] = i
+# B64 = [-1] * 256
+# for i, c in enumerate('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'):
+#    B64[ord(c)] = i
+B64 = [-1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
```

Comment:
Is this the same as what is generated by lines 237-239?

Reply:
Yes basically, I reverted to just using the enumeration